pex/bin/pex.py

```diff
@@ -325,6 +326,29 @@ def configure_clp_pex_environment(parser):
         "#!. This overrides the default behavior, which picks an environment Python "
         "interpreter compatible with the one used to build the PEX file.",
     )
+    group.add_argument(
+        "--shboot",
+        "--sh-boot",
+        "--bash-boot",
```

Comment:
I went a bit crazy with option names. The `sh` variants make sense. The `--bash-boot` is what was originally in my head though and is fun when spoken in a ~German way with zest.

Reply:
Crazy indeed :P 

Any reason to why you failed to prune this list down, or otherwise motivating factor to keep them all ?

Reply:
+1 on considering removing `bash`.

Is there precedent for `shboot` affordance rather than `sh-boot`?

Reply:
No motivating factor to keep `--[no-]shboot`. The `--bash-boot` I just like saying. I'll kill the `--[no-]shboot` variation here in a moment. They're harmless, but I'd also like to take the CI burn opportunity to reverse the fallback python search order such that newer compatible Pythons are selected 1st when populating from Pex's own IC.

Reply:
Ok, you all are no fun!. I'll kill both.